sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/redis/JedisRedisCheckpointStoreBuilder.java

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.eventhubs.checkpointstore.redis;
+
+import com.azure.core.credential.AzureKeyCredential;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+final class JedisRedisCheckpointStoreBuilder {
+    private String hostname = "";
+    private String key;
+    private JedisPool jedisPool;
+    private int operationTimeout = 1000;
+    private int connectionTimeout = 5 * operationTimeout;
+    private int maxPoolConnections = 200;
+    private int maxPoolIdleConnections = 100;
+    private int minPoolIdleConnections = 50;
+    private int maxWait = operationTimeout;
+
+    /**
+     * Sets the hostname for the Azure Redis Cache associated with the CheckpointStore
+     * @param endpoint a String representing the hostname for Azure Redis Cache
+     * @return a JedisRedisCheckpointStoreBuilder instance
+     */
+    public JedisRedisCheckpointStoreBuilder endpoint(String endpoint) {
+        this.hostname = endpoint;
+        return this;
+    }
+    /**
+     * Sets the password for the Azure Redis Cache associated with the CheckpointStore
+     * @param key an AzureKeyCredential representing the key for Azure Redis Cache
+     * @return a JedisRedisCheckpointStoreBuilder instance
+     */
+    public JedisRedisCheckpointStoreBuilder credential(AzureKeyCredential key) {
+        this.key = key.getKey();
+        return this;
+    }
+    /**
+     * Builds an instance of the JedisRedisCheckpointStore class
+     * @return JedisRedisCheckpointStore object
+     */
+    public JedisRedisCheckpointStore build() {
+        boolean BLOCK_POOL_WHEN_EXHAUSTED = true;
```

Comment:
Does this pass checkstyles?

Reply:
Generally, there are naming conventions we use to keep our code consistent... We have a variation on this.. but in general, the naming convention holds true.

https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s5-naming

Reply:
I'll just change these, but somehow they passed during CheckStyle.